File: delivery-service/store-admin/src/main/java/org/delivery/storeadmin/domain/userorder/controller/model/UserOrderDetailResponse.java
package org.delivery.storeadmin.domain.userorder.controller.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.delivery.storeadmin.domain.storeMenu.controller.model.StoreMenuResponse;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserOrderDetailResponse {

    private UserOrderResponse userOrderResponse;

    private List<StoreMenuResponse> storeMenuResponseList;
}

File: delivery-service/api/src/main/kotlin/org/delivery/api/domain/userOrder/controller/model/UserOrderDetailResponse.kt
package org.delivery.api.domain.userOrder.controller.model

import org.delivery.api.domain.store.controller.model.StoreResponse
import org.delivery.api.domain.storeMenu.controller.model.StoreMenuResponse

data class UserOrderDetailResponse (

   val userOrderResponse: UserOrderResponse? = null,
   val storeResponse: StoreResponse? = null,
   val storeMenuResponseList: List<StoreMenuResponse>? = null
)